### PR TITLE
add environment test feature

### PR DESCRIPTION
**Pull request by yuichi10**
## Context

There are no functional tests for Environment functions. The function is a very core feature, thus it is good to check by acceptance test. 

## Objective

- Add acceptance test for environments function.
  - Already created the test repository ([functional-environments](https://github.com/screwdriver-cd-test/functional-environments))

## License
I confirm that this contribution is made under a BSD license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: features/environments.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+@ignore-for-deploy
+Feature: Envitonments
+
+    Screwdriver exports a set of environment variables that you can rely on during build runtime
+
+    Rules:
+        - Environment variables should be available in a job
+
+    Background:
+        Given the pipeline with setting environment variables should be exist
```
Comment (tkyi) [inline features/environments.feature:11]:
```suggestion
        Given an existing pipeline with setting environment variables
```

File: features/step_definitions/environments.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    {
+        tags: '@environments'
+    },
+    function hook() {
+        this.repoName = 'functional-environments';
+    }
+);
+
+Given(
+    /^the pipeline with setting environment variables should be exist$/,
```
Comment (tkyi) [inline features/step_definitions/environments.js:19]:
```suggestion
    /^an existing pipeline with setting environment variables$/,
```

File: features/environments.feature [hunk 3]
```diff
@@ -0,0 +1,15 @@
+@environments
+@ignore-for-deploy
```
Comment (tk3fftk) [inline features/environments.feature:2]: Is this annotation needed?

File: features/environments.feature [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    Screwdriver exports a set of environment variables that you can rely on during build runtime
+
+    Rules:
+        - Environment variables should be available in a job
+
+    Background:
+        Given an existing pipeline with setting environment variables
+
+    Scenario: Check Environment can be used
+        When the "main" job with setting environment variables is started
```
Comment (tk3fftk) [inline features/environments.feature:14]: Should we make these steps more precisely?

File: features/environments.feature (hunk 3 above)
Comment (yuichi10) [reply features/environments.feature]: I deleted the annotation.

File: features/environments.feature (hunk 4 above)
Comment (yuichi10) [reply features/environments.feature]: I fix the sentences.

**Review by tk3fftk — APPROVED** [review]
:+1:

**Comment by yuichi10** [comment]
@tkyi 
I fixed this PR. Would you review this PR again?

**Review by jithine — APPROVED** [review]

**Review by tkyi — APPROVED** [review]

**Pull request merged**